src/main/java/domain/expression/Expression.java

```diff
@@ -0,0 +1,33 @@
+package domain.expression;
+
+import domain.expression.operator.Operator;
+
+public class Expression {
+
+    private int left;
+    private int right;
+    private Operator operator;
```

Comment:
```suggestion
    private final int left;
    private final int right;
    private final Operator operator;
```
Expression을 불변 객체로 만들 수도 있겠네요 🙂